go.mod

```diff
@@ -7,9 +7,9 @@ require (
 	github.com/google/go-cmp v0.5.5
 	github.com/onsi/ginkgo v1.16.4
 	github.com/onsi/gomega v1.13.0
-	github.com/open-cluster-management/addon-framework v0.0.0-20210621074027-a81f712c10c2
-	github.com/open-cluster-management/go-template-utils v1.3.0
-	github.com/open-cluster-management/governance-policy-propagator v0.0.0-20211012174109-95c3b77cce09
+	github.com/stolostron/addon-framework v0.0.0-20210621074027-a81f712c10c2
```

Comment:
I suggest you revert the org change for any dependencies and have a separate PR to update after dependencies have been updated.

Reply:
Good point. We'll be able to move ahead faster that way. I'll update it.